source/components/Autocomplete.js

```diff
@@ -1,25 +1,35 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
```

Comment:
We are enforcing single quotes in Daedalus and i think it would be good to keep it that way in all projects.

Reply:
@DominikGuzei
I worked on this PR a couple weeks ago before I joined and didn't eslint and auto formatting configured. At the time I also didn't know the naming convention for PR's, but Nikola set me straight on that. 

*Should I continue to add a description of the PR as I've done here? I try to make my commits small to supplement the code review, (I know you can't tell yet... 😅), but I don't want to create even more noise if the PR description is not useful or unnecessary. I try to make it like a checklist. *

Also, you are exactly right about the code formatting being automatic on save. That being said, in [PR 58](https://github.com/input-output-hk/react-polymorph/pull/58), I ported over the .eslintrc file from Daedalus and applied those rules to the auto format plugin that I'm running. I noticed it enforced double quotes as well as shortened lines of code which create a less legible format like you've pointed out in your comment below about component story titles. I actually prefer all the formatting conventions you've pointed out here and want to start using those asap, but I think we are going to have this same issue in PR 58 because I ran auto format on the entire library thinking I was applying the correct rules from the Daedalus .eslintrc file. There is going to be a ton of noise there and it won't even serve a purpose because it was obviously enforcing the wrong rules.

*As a preventative measure against repeating this, would you please give me a rundown on your text editor and the relevant plugins you're using for auto format (if you even use that) and also what you're using to displaying eslint/flow warnings?* Having a PR that is 70% formatting noise basically foils the code review. I've run into this same problem in the past while reviewing and I was pretty frustrated, so I don't want to keep guessing on whether my configuration is installed properly and repeat this.

@nikolaglumac *Before you review [PR 58](https://github.com/input-output-hk/react-polymorph/pull/58), maybe I should first configure auto format correctly and fix what appears to be a gambit of Daedalus anti-conventions before its merged with 149?* I can do this tonight after I get in touch with one of you on the plugin situation.

Reply:
@MarcusHurney i think highest priority is to revert all the formatting changes in this PR and get it merged so we can do the reviews on your other PRs!